Javacsript/mobile-menu.js

```diff
@@ -0,0 +1,25 @@
+const menuToggle = document.querySelector('.Sidebar-class');
+const mobileMenu = document.querySelector('.mobile-menu');
+const menuItems = document.querySelectorAll('.mobile-menu-items li a');
+const closeBtn = document.querySelector('.close-btn');
+// const header = document.querySelector('header');
```

Comment:
- [ ] Let's kindly remove commented codes. We shouldn't commit them to our repo. They make your project look messy. If you need to review the previous version of your code, you can always use git history.